### ISSUE TITLE
[Problem] Playlist is removed when you return to the page of the video in the corner.

### ISSUE THREAD

**Issue by JO-Arden925** [Circularify/Queue-n-Play#1]
**Platform:** Windows 8.1
**Chrome Version:** 50.0.2661.102 m

**Steps to create problem:** 
1. Open a video
2. Start browsing so video is in the lower right-hand corner
3. Add videos to the queue 
4. Press the button that brings you to the page of the video playing in the corner 
(I circled it and pointed at it with a red arrow in the picture below)
![queue n play](https://cloud.githubusercontent.com/assets/19352901/15261330/d05bb940-1929-11e6-90da-eecf3d269004.png)

**Comment by martijnlentink** [comment]
Hi there Arden, thank you for taking the time to write us a bug report on Queue 'n Play. We hope you'll enjoy it as much as we do. As you may have noticed the extension is still quite new and has some rough edges. In the near future we will try to fix these bugs and move toward a more stable solution.

As on your bug report; you noticed that the queue disappeared during the navigation to a video, this is true. But the actual queue still exists and is just not visible anymore, Queue n Play will still queue coming videos even if the queue is not visible. We understand that this is not always convenient, but focussed really on the importance of adding video's to the queue while navigation.

In summary; QnP is new and still has some caveats, we will be looking at a solution where your queue is always visible to the user and for now you should know that QnP is still working, only not directly showing your queued videos.

Thank you! The circularify team

**Comment by dickwolff** [comment]
This is implemented in the next release.
